modules/tests/support/model_based/IBCTests.cfg

```diff
@@ -0,0 +1,8 @@
+CONSTANTS
```

Comment:
Not clear what is the difference b/t this file and `modules/tests/support/model_based/IBC.cfg`.

Reply:
Tried to improve the README explaining this in 2e5d6de82f69a14046b8555f95c706701b140415.